### PR TITLE
add cli plugins --list option

### PR DESCRIPTION
**Pull request by stafyniaksacha**

### CODE REVIEW

File: bin/commands/plugins.js [hunk 1]
```diff
@@ -88,8 +106,10 @@ module.exports = function pluginsManager (plugin, options) {
       .then(() => {
         release();
 
-        if (!plugin) {
-          console.log(clcOk('███ kuzzle-plugins: Plugins installed'));
+        if (options.install) {
```
Comment (ballinette) [inline bin/commands/plugins.js]: avoid nested if-then-else if not needed.
Here, we can just write 
```
if (options.install && !plugin) {
// (...)
}
```

Or: 
```
if (options.install) {
  if (plugin) {
    console.log(clcOk('███ kuzzle-plugins: Plugin ' + plugin ' installed'));
  }
  else {
    console.log(clcOk('███ kuzzle-plugins: Plugins installed')); 
  }
}
```

File: bin/commands/plugins.js [hunk 2]
```diff
@@ -427,27 +443,29 @@ function checkOptions(plugin, options) {
     process.exit(1);
   }
 
-  // --install is the only option working without specifying a plugin name
-  if (!plugin && !options.install) {
-    console.error(clcError('A plugin [name] is required for this operation'));
-    process.exit(1);
-  }
+  // --install and --list are the only options working without specifying a plugin name
+  if (!options.list) {
```
Comment (ballinette) [inline bin/commands/plugins.js]: This code should be improved : 
* --install and --list are mutually exclusive (what is not checked yet)
* the condition "if (!plugin && !options.install && !options.list)" should not be nested.

Improvement proposal : 
```
if (!plugin && !options.install && !options.list) {
  console.error(clcError('A plugin [name] is required for this operation'));
  process.exit(1);
}

if (options.install && options.list) {
  console.error(clcError('Options --install and --list are mutually exclusive));
  process.exit(1);
}

// then put the rest of the code, unchanged
```

**Comment by ballinette** [comment]
despite my nitpicking remarks, +1

File: bin/commands/plugins.js (hunk 1 above)
Comment (stafyniaksacha) [inline bin/commands/plugins.js]: Done

**Comment by scottinet** [comment]
+1

**Pull request merged**